src/screens/SendAddressInput.js

```diff
@@ -5,73 +5,71 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import { KeyboardAvoidingView, View } from 'react-native';
 import { t } from 'ttag';
+import { Network } from '@hathor/wallet-lib';
+import { useSelector } from 'react-redux';
 
 import NewHathorButton from '../components/NewHathorButton';
 import SimpleInput from '../components/SimpleInput';
 import HathorHeader from '../components/HathorHeader';
 import { getKeyboardAvoidingViewTopDistance, validateAddress } from '../utils';
 import OfflineBar from '../components/OfflineBar';
 
-class SendAddressInput extends React.Component {
+export const SendAddressInput = ({ route, navigation }) => {
```

Comment:
Can you use the `useNavigation` and `usRoute` hooks instead of getting it from the props?

Reply:
Sure, here it is: https://github.com/HathorNetwork/hathor-wallet-mobile/pull/376/commits/b78e69bae51c737db95c84948a3125638d8e2da0